### ISSUE TITLE
Unable to send unsecured text messages from signal app

### ISSUE THREAD

**Issue by bitcoinbabe** [WhisperSystems/Signal-Android#4854]
Issue started about a week ago on my Samsung Galaxy S5. It no longer sends unsecured messages to people who dont have the app.. It just gets stuck in a pending state indefinitely. signal > signal messages work fine, and I can receive regular SMS/MMS's fine.

**Comment by moxie0** [comment]
Please reproduce your issue and capture a debug log immediately after:
https://github.com/WhisperSystems/Signal-Android/wiki/Submitting-useful-bug-reports

**Comment by bitcoinbabe** [comment]
https://gist.github.com/88df4a390b48a55de577

**Comment by moxie0** [comment]
this log looks like you're attempting to send an MMS, is that what you're having problems with, not SMS?

**Comment by bitcoinbabe** [comment]
No Im trying to send an SMS

**Comment by bitcoinbabe** [comment]
![screenshot_2015-12-10-23-12-54](https://cloud.githubusercontent.com/assets/16239654/11715144/b7daf83c-9f93-11e5-95bb-658e394e000b.png)

**Comment by bitcoinbabe** [comment]
dont mind the jibberish, Im just trying to send test messages to my friends phone

**Comment by moxie0** [comment]
Have you tried restarting your phone? There's no attempt to send an SMS in the log, which means that the device is reporting cellular service is unavailable.

**Comment by bitcoinbabe** [comment]
Ive restarted the phone multiple times, uninstalled and reinstalled the app, and still the issue exists. As I said this has been on going for a week.

**Comment by moxie0** [comment]
Is your device rooted, running a non-stock ROM, xposed, or any other weird shit?

**Comment by bitcoinbabe** [comment]
Nope its 100% stock.

**Comment by moxie0** [comment]
Strange, for some reason your device is reporting that it has no mobile service. There's unfortunately not much we can do if it lies to us.

**Comment by bitcoinbabe** [comment]
If that were the case then technically I shouldnt be able to message signal>signal if there is no service? I can still send to signal devices with my wifi off...

Also I can still receive regular SMS and MMS?

**Comment by agrajaghh** [comment]
Is it a dual sim phone? related to / duplicate of #555?

**Comment by bitcoinbabe** [comment]
It is a dual sim phone, but I only have 1 sim card in the phone. Havent used the 2nd SIM slot in over 6 months.

**Comment by moxie0** [comment]
@bitcoinbabe The device reports two different states: network service and cellular service. Signal messages require network service, while SMS requires cellular service.

**Comment by bitcoinbabe** [comment]
I just tried sending a photo via unsecured. That went through!

**Comment by moxie0** [comment]
@bitcoinbabe Makes sense, MMS requires network service rather than cellular service.

**Comment by atomichorror** [comment]
I was able to solve identical problem on my Xiaomi Redmi 2. I have two SIM cards in slots: slot 1 - SIM card with limitless internet, slot 2 - SIM dedicated for phone calls and sending messages. I read about it and it seems like dual SIM phones are sligtly different when it comes to software which controls SIM management. Personally, I swapped SIMs in slots and everything works fine now, so it looks like you have to have the SIM from which you call in the slot identified as the first. Previously I was able to receive messages only. Try it yourself, it may help!

**Comment by riyapenn** [comment]
Closing as dual SIM is now supported #555.

**Comment by Officermick** [comment]
YES atomichorror,
I found the same solution for the problem!
I have a Umidigi Z1 PRO with  dual SIM slots with one SIM card installed it.
I put the SIM card in the 2nd slot and NOW I can  receive unsecured SMS  threw Moblie Txt from my Mobile network.

I moved the SIM card from SIM Slot 1 to SIM Slot 2 in my Umidigi Z1 PRO.

Now, I don't have  to use a 2nd TXT app to receive TXTs from people who don't use the signal app.

Furthermore, We should encourage everyone to use the signal app and encryption apps (like Whatsapp) as the eyes of Big Brother  are watching us. 
This will only get worse as Julian Assange & Edwards Snowden has warned us.

Death to Big Brother!!!!!!
